features/user.feature

```diff
@@ -209,6 +209,14 @@ Feature: Manage WordPress users
   Scenario: Managing user roles
     Given a WP install
 
+    When I run `wp user add-role 1`
```

Comment:
@danielbachhuber On my local, this test is failing, even though the feature works.  Any pointers as to what I did wrong here?

Reply:
```suggestion
    When I try `wp user add-role 1`
```

@shawnhooper It should be `When I try`, not `When I run`. 'try' is for error cases, 'run' is for success cases.